src/main/java/seedu/duke/command/misc/RecipePossibleCommand.java

```diff
@@ -7,19 +7,36 @@
 import seedu.duke.recipe.Recipe;
 import seedu.duke.recipe.RecipeList;
 
+/**
+ * Represents the "/recipe possible" command.
+ */
 public class RecipePossibleCommand extends ExecutableCommand {
-
-    private boolean hasEnoughIngredient(Ingredient recipeIngredient, IngredientList fridgeIngredients) {
-        String recipeIngredientName = recipeIngredient.getName();
-        double recipeIngredientQty = recipeIngredient.getQuantity();
-        for (Ingredient fridgeIngredient : fridgeIngredients.getIngredients()) {
-            if (fridgeIngredient.getName().equals(recipeIngredientName) &&
-                    fridgeIngredient.getQuantity() >= recipeIngredientQty) {
+    /**
+     * Check if an ingredientList has a sufficient amount of an ingredient.
+     *
+     * @param ingredient     the ingredient to look for
+     * @param ingredientList the list of ingredients to check in
+     * @return true if the list of ingredients have sufficient quantity of that ingredient, false otherwise
+     */
+    private boolean hasEnoughIngredient(Ingredient ingredient, IngredientList ingredientList) {
+        String recipeIngredientName = ingredient.getName();
+        double recipeIngredientQty = ingredient.getQuantity();
+        for (Ingredient fridgeIngredient : ingredientList.getIngredients()) {
+            if (fridgeIngredient.getName().equals(recipeIngredientName)
+                    && fridgeIngredient.getQuantity() >= recipeIngredientQty) {
                 return true;
             }
         }
         return false;
     }
```

Comment:
Could this be refactored to avoid deep nesting?

Reply:
We could explore using a hash map instead of an array for our IngredientList, RecipeList. Shall we KIV this first and refactor in V2.0 if we decide to continue with the use of array?

Reply:
Yes we could always discuss more efficient ways of storing our data in future meetings